src/Compilers/Core/Desktop/EncodedStringText.cs

```diff
@@ -23,7 +23,7 @@ internal static class EncodedStringText
         /// from <see cref="SourceText.From(Stream, Encoding, SourceHashAlgorithm, bool)"/> in two ways:
         /// 1. It attempts to minimize allocations by trying to read the stream into a byte array.
         /// 2. If <paramref name="defaultEncoding"/> is null, it will first try UTF8 and, if that fails, it will
-        ///    try <see cref="Encoding.Default"/>.
+        ///    try <see cref="Encoding.ASCII"/>.
```

Comment:
I don't see how we could possibly succeed at getting ASCII but failing to get UTF8, since they're backwards compatible.